pandas/core/ops/mask_ops.py

```diff
@@ -37,11 +37,13 @@ def kleene_or(
         The result of the logical or, and the new mask.
     """
     # To reduce the number of cases, we ensure that `left` & `left_mask`
-    # always come from an array, not a scalar. This is safe, since because
+    # always come from an array, not a scalar. This is safe, since
     # A | B == B | A
     if left_mask is None:
         return kleene_or(right, left, right_mask, left_mask)
 
+    if not isinstance(left, np.ndarray):
```

Comment:
can remove the following assert now? (and elsewhere)

Reply:
Removed them, I thought mypy might still need them.